### ISSUE TITLE
Show Email button in footer

### ISSUE THREAD

**Issue by cassidyjames** [elementary/appcenter#725]
Sometimes directly sending an email to a developer is appropriate, i.e. to provide feedback. We could use the email address associated with the GitHub account to add an "Email" button to the footer, which would pop open the Mail compose dialog with a pre-filled recipient and subject.

Implementation-wise, I don't see an email address in the AppStream spec, so if we wanted to do it via that route, we could use an `x-appcenter-email` tag or something, either entered manually by a developer if they want to use this feature, or automatically by Houston if we decide this is something we want to provide for all apps.

Google, for example, requires an email address for apps on Android and displays a "Send email" item in the Google Play Store footer alongside the "Visit website," "Privacy policy," etc. items.

**Comment by cassidyjames** [comment]
Fwiw, this feature was directly inspired by a developer asking for user reviews or some other way for users to easily provide feedback to developers.

**Comment by danrabbit** [comment]
I think we might need to either redesign the whole footer or think about a different way to display this info. The footer is pretty crowded as is:

![screenshot from 2018-06-14 14 32 45 2x](https://user-images.githubusercontent.com/7277719/41439478-d5411b1c-6fdf-11e8-98ee-13e8ea4fc711.png)

**Comment by stsdc** [comment]
Maybe it will be better to add email option to Author/Developer view instead of Application view?

**Comment by cassidyjames** [comment]
@danrabbit that's fair. One thing we could do hierarchy-wise is split it into links and in-app actions, though that's not super clear at a glance. I'll think about it. :smile:

**Comment by cassidyjames** [comment]
@stsdc yeah I was wondering about that, too, but we don't currently have a good concept of developers inside AppCenter. And theoretically if it's a field the developer sets, it could differ per app. If we pull it from GitHub, it would be the same for all of their apps, however.
